.dockerignore

```diff
@@ -5,3 +5,19 @@ CTFd/*.db
 CTFd/uploads/**/*
 .ctfd_secret_key
 .data
+
+.codecov.yml
+.dockerignore
+.git
+.github
+.gitignore
+.prettierignore
+.travis.yml
+*.md
+Dockerfile
+Vagrantfile
+development.txt
+docker-compose.yml
+docs
+setup.cfg
+tests
```

Comment:
I don't really agree with the premise of the container should only contains what it needs to run. What if I develop CTFd using Docker? 

Will just putting `.git` in here prevent it from being copied during the COPY command?